sql/postgres/inspect.go

```diff
@@ -228,7 +232,7 @@ func (i *inspect) addColumn(t *schema.Table, rows *sql.Rows) error {
 
 func columnType(c *columnDesc) schema.Type {
 	var typ schema.Type
-	switch t := c.typ; strings.ToLower(t) {
+	switch t := strings.Split(c.typ, "(")[0]; strings.ToLower(t) {
```

Comment:
Do you have examples in which the `data_type` returned with `(`? I'm pretty sure the test examples are not correct (e.g. `timestamp(4)`).

Reply:
I'll check